.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,4 +19,7 @@
  https://kubernetes.io/docs/contribute/new-content/overview/#choose-which-git-branch-to-use
  for advice.
 
+ Use the next release branch, "dev-x.xx", if you're documenting a feature for
+ a release. If applicable, please also include links to k/k and k/enhancement.
```

Comment:
What are you asking PR authors to do?
I think there is an explanation about contributing docs for new features in the docs, https://kubernetes.io/docs/contribute/new-content/new-features/
Could you link to the documentation instead?
Or you could update the docs with information about linking docs PRs to pull requests
in the `kubernetes/kubernetes` and `kubernetes/enhancement` repositories.